COLLADAStreamWriter/include/COLLADASWConstants.h

```diff
@@ -137,6 +142,19 @@ namespace COLLADASW
         static const String CSW_ELEMENT_MATERIAL;
         static const String CSW_ELEMENT_MATRIX;
         static const String CSW_ELEMENT_MESH;
+		static const String CSW_ELEMENT_CONVEX_MESH;
```

Comment:
Since this is an opensource project, could you use the same tabulation (whitespaces/tab) that was in the original file?

(maybe it is, but github display it differently, so I'd rather ask )